app/test/frontend/golden/my_liked_packages.html

```diff
@@ -74,7 +74,7 @@ <h1 class="_visuallyhidden">Dart pub</h1>
     <div class="_banner-bg">
       <div class="container">
         <div class="small-banner">
-          <form class="search-bar" action="/packages">
+          <form class="search-bar" action="/my-packages">
```

Comment:
Same question here

Reply:
Yes, we are on the "My liked packages" page, and when I search for something, I'd expect it to be inside my liked package, or as a fallback, inside my packages.